backend/CircleForms/Services/Database/PostRepository.cs

```diff
@@ -156,6 +180,11 @@ public async Task AddAnswer(string postId, Answer entry)
         }
     }
 
+    private static long ToUnixTimestamp(DateTime time)
```

Comment:
Move to an extension class

Reply:
resolve problems at the rate of occurring

Reply:
You're creating unnecessary tech debt

Reply:
It's not a tech debt, it's a reduction of abstractions. Currently we don't need to use ToUnixTimestamp in other scenarios, so what's the point